CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Changes
 
+* [#7107](https://github.com/rubocop-hq/rubocop/issues/7107): Fix parentheses offence for numeric arguments with an operator in `Style/MethodCallWithArgsParentheses`. ([@gsamokovarov][])
```

Comment:
I think this should be under "Bugs fixed".

Reply:
Oh, of course!